openapi.yaml

```diff
@@ -6726,6 +6726,48 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-delete 12345 456
+  /lke/clusters/{clusterId}/pools/{poolId}/recycle:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to this Node Pool is attached to.
```

Comment:
This reads a lil funny, I think the "to" can be removed, like
"ID of the Kubernetes cluster this Node Pool is attached to."

Reply:
fixed